apps/web/pages/new-booker/[user]/[type].tsx

```diff
@@ -123,7 +127,9 @@ async function getUserPageProps(context: GetServerSidePropsContext) {
 
   let booking: GetBookingType | null = null;
   if (rescheduleUid) {
-    booking = await getBookingByUidOrRescheduleUid(`${rescheduleUid}`);
+    booking = await getBookingForReschedule(`${rescheduleUid}`);
+  } else if (bookingUid) {
+    booking = await getBookingForSeatedEvent(`${bookingUid}`);
```

Comment:
The logic here changed a little bit, if booking had bookingUid it was completely ignored. From now receiving it means that we have a seated event.
Receiving rescheduleUid means that we either received bookingUid or bookingSeatUid.

Reply:
I separated in two functions, bookingForReschedule returns current user data, while getting a booking from a seated event should only return attendees.length.